Examples/DicomSeriesFromArray/Documentation.rst

```diff
@@ -0,0 +1,26 @@
+.. _lbl_dicom_series_from_array:
+
+Dicom Series From Array
+==============================
+
+Overview
+--------
+This example illustrates how to write a DICOM series from an array in Python
+
+Generating an array is done using a simple random number generator for this case but can come from other sources.
+
+Writing the 3D image as a DICOM series is done by configuring the meta-data dictionary for each of the slices and then writing it in DICOM format. In our case we copy some of the meta-data from the original dictionaries which are available from the series reader. We then set some additional meta-data values to indicate that this series is derived from the original acquired data. Note that we write the intensity values as is and thus do not set the rescale slope (0028|1053), rescale intercept (0028|1052) meta-data dictionary values.
```

Comment:
Please remove the reference to copying "some meta-data" as this is not really what happens.